lib/init.js

```diff
@@ -58,6 +59,7 @@ export default {
 
         const messages = [];
         let match = parseRegex.exec(result);
+        let globalsOmitted = atom.config.get('linter-moonscript.omitGlobalCheck');
```

Comment:
Please observe this setting [like this](https://github.com/AtomLinter/linter-moonscript/blob/772d30b15c4d2780fcfb76c48e6ba7a1246420c0/lib/init.js#L18), there's no need to go through the complexity of determining what it is and what the current value is on every call 😉.